packages/cf-component-loading/src/Loading.js

```diff
@@ -3,8 +3,8 @@ const React = require('react');
 class Loading extends React.Component {
   render() {
     return (
-      <div className="cf-loading" role="status">
-        <span className="cf-loading__label">Loading</span>
+      <div className="cf-icon--loading" role="status">
```

Comment:
I would keep `<Loading/>` as well because it should abstract away the a11y details. Just depend on icon